docs/development/creating-your-first-command.md

```diff
@@ -18,7 +18,7 @@ public class CommandVelocity extends Command {}
 
 ## Using Annotations
 
-Commands in GoMint are based on annotations and injection. There are two required annotations that each command must have, but the full list of annotations for commands are below.
+Commands in GoMint are based on annotations and injection. There are two required annotations that each command must have, but the full list of annotations for commands are below. GoMint will detect all classes in your plugin which extend `Command` and have at least the required annotations and will automatically create an instance and register it.
```

Comment:
Maybe this should also state on how to register custom built commands with `Plugin.registerCommand(Command);`

Reply:
Oh i wanted to add this at the bottom where I explained the custom command description actually but I forgot it. I think it should be down there and not here at the top.
Will add it tomorrow.